public/pages/CreateTrigger/components/CompositeTriggerCondition/ExpressionBuilder.js

```diff
@@ -244,21 +245,16 @@ const ExpressionBuilder = ({
       data-test-subj={`${formikFullFieldName}_${triggerIndex}_${idx}_options`}
     >
       <EuiFlexItem grow={false}>
-        <EuiComboBox
-          style={{ width: '150px' }}
-          singleSelection={{ asPlainText: true }}
-          data-test-subj={`condition-combobox-${triggerIndex}-${idx}`}
+        <EuiSelect
           compressed
-          selectedOptions={[
-            {
-              label: expression.description,
-              description: expression.description,
-            },
-          ]}
-          onChange={(selection) => changeCondition(selection, expression, idx, form)}
+          style={{ width: '100px' }}
+          data-test-subj={`condition-combobox-${triggerIndex}-${idx}`}
```

Comment:
Nitpick: This `data-test-subj` could be changed as this component is no longer a `combobox`; but this isn't blocking.

Reply:
Fixed